frontend/src/metabase/dashboard/components/DashCard.jsx

```diff
@@ -79,13 +81,13 @@ export default class DashCard extends Component {
             .map(card => ({
                 ...getIn(dashcardData, [dashcard.id, card.id]),
                 card: card,
-                duration: cardDurations[card.id]
+                isSlow: (slowCards[card.id] && card.query_average_duration && (card.query_average_duration > DATASET_USUALLY_FAST_THRESHOLD))
```

Comment:
To make it a little cleaner you could also compute

```javascipt
isUsuallyFast: card.query_average_duration < DATASET_USUALLY_FAST_THRESHOLD
```

here and use that below.